client/app/lib/components/buttons/EditButton.tsx

```diff
@@ -0,0 +1,14 @@
+import { IconButton, IconButtonProps } from '@mui/material';
+import Edit from '@mui/icons-material/Edit';
+
+interface Props extends IconButtonProps {
+  onClick: (event: any) => void;
```

Comment:
I think can type the event as `SyntheticEvent` or something like that.

Reply:
Got it!